templates/certified/components/component-details.html

```diff
@@ -0,0 +1,34 @@
+{% extends "templates/base.html" %}
+
+{% block title %}Ubuntu certified component: {{ component.vendor_name }} {{ component.model }}{% endblock %}
+
+{% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block outer_content %}
+
+<div class="row">
```

Comment:
No need to wrap the entire page in a row, each section has it's own row already defined